utils/rpms/daos.spec

```diff
@@ -7,7 +7,7 @@
 
 Name:          daos
 Version:       1.1.0
-Release:       6%{?relval}%{?dist}
+Release:       7%{?relval}%{?dist}
```

Comment:
This will need to be updated -- I just landed a PR this morning that took release 7.

Reply:
Ahh I just rebased last night that had it go to 7. I'll bump it to 8.